src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,19 @@
+@import "./../../utils/variables.scss";
+
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 20px 0;
+  border-radius: $-border-radius;
+  background-color: $-color-background-todo;
```

Comment:
Consider using a more descriptive background color instead of `$-color-background-todo`. For example, `$-color-background-todo-info`. 🎨